src/Mapper/Default_Content_Mapper.php

```
@@ -0,0 +1,691 @@
+	 *     return is_wp_error( $user_id ) ? 1 : $user_id;
+	 * }
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return int The author user ID.
+	 */
+	public function get_author(): int {
+		// ============================================
+		// ADD YOUR OWN AUTHOR EXTRACTION LOGIC HERE
+		// ============================================
+		$content = $this->content_scrapper->get_content();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$content'. (Unused Code Rules)

(UnusedLocalVariable)